bin/charm-env

```diff
@@ -43,7 +43,7 @@ find_charm_dirs() {
             return
         fi
         # shellcheck disable=SC2126
-        non_subordinates="$(grep -L 'subordinate:.*true' "$agents_dir"/unit-*/charm/metadata.yaml | wc -l)"
+        non_subordinates="$(grep -L 'subordinate"\?:.*true' "$agents_dir"/unit-*/charm/metadata.yaml | wc -l)"
```

Comment:
are you sure this is correct? I would expect that " would need to be escaped.

Reply:
It's inside a single-quoted string.

Reply:
FWIW, tested with ubuntu (locally built with this layer) and nrpe (from the charm store).  I was never able to replicate the import error, but the subordinate disambiguation worked as expected.